File: pythonclass/Chapter3/java/Question8.java
import java.util.Scanner;
public class Question8 {
	public static void main(String[] args) {
	Scanner input = new Scanner(System.in);

int total = 0;
int product = 0;
double average =0;
int[] numbers = new int[3];


for (int count=1; count <=3; count++){
	System.out.print("Enter a number");
	int userInput = input.nextInt();
	System.out.print("you entered " + userInput);
	number[] = userInput;
	total += userInput;
	product *= userInput;
	average = total /3;}

System.out.println("the total of the numbers" + total);
System.out.println("the product of the numbers" + product);
System.out.println("the average of the numbers" + average);

int smallestNumber = Integer.MIN_VALUE;

int largestNumber = Integer.MAX_VALUE;

System.out.println("The smallest number is " + smallestNumber);
System.out.println("The largest number is " + largestNumber);
}
}

File: pythonclass/Chapter3/python/question8.py
total = 0
average = 0
product = 0

numbers =[]
for count in range(3):
    user_input= int(input('Enter a number: '))
    print('you entered', user_input)
    numbers.append(user_input)
    total += user_input
    average = total / 3
    product *=user_input
print('the product is ', product)
print('the total is ', total)
print('the average is ', average)

minimum_number = min(numbers)
maximum_number = max(numbers)
print('the smallest number is ', minimum_number, 'the largest number is ', maximum_number)
